web/app/js/components/TabbedMetricsTable.jsx

```diff
@@ -167,7 +164,7 @@ export default class TabbedMetricsTable extends React.Component {
     }
     this.setState({ pendingRequests: true });
 
-    this.api.fetch(this.state.metricsUrl.ts)
+    this.api.fetchMetrics(this.state.metricsUrl.ts)
```

Comment:
a few lines below this, `10 MINUTE HISTORY` is hard-coded.

Reply:
omg, I thought I got all of those. thanks for catching!